refarch-backend/src/main/resources/application-local.yml

```diff
@@ -3,12 +3,6 @@ server:
 spring:
   # Spring JPA
   jpa:
-    hibernate:
-      # always drop and create the db should be the best
-      # configuration for local (development) mode. this
-      # is also the default, that spring offers by convention.
-      # but here explicite:
-      ddl-auto: create-drop
```

Comment:
I'd suggest changing this to `validate` instead of completely removing it.

Reply:
done